board/freescale/mx6sxsabresd/mx6sxsabresd.c

```diff
@@ -265,9 +291,76 @@ static int setup_lcd(void)
 	/* Set Brightness to high */
 	gpio_request(IMX_GPIO_NR(6, 4), "lcd_bright");
 	gpio_direction_output(IMX_GPIO_NR(6, 4) , 1);
+}
 
-	return 0;
+static void enable_lvds(struct display_info_t const *dev)
+{
+	int ret;
+
+	ret = enable_lcdif_clock(dev->bus, 1);
+	if (ret) {
+		printf("Enable LCDIF clock failed, %d\n", ret);
+		return;
+	}
+	ret = enable_lvds_bridge(dev->bus);
+	if (ret) {
+		printf("Enable LVDS bridge failed, %d\n", ret);
+		return;
+	}
+
+	imx_iomux_v3_setup_multiple_pads(lvds_ctrl_pads,
+							ARRAY_SIZE(lvds_ctrl_pads));
+
+	/* Enable CABC */
+	gpio_request(IMX_GPIO_NR(4, 18), "CABC enable");
+	gpio_direction_output(IMX_GPIO_NR(4, 18) , 1);
```

Comment:
U-Boot patch checker error: space prohibited before that ','
```suggestion
	gpio_direction_output(IMX_GPIO_NR(4, 18), 1);
```